src/Controllers/WhoopsController.php

```diff
@@ -16,7 +16,7 @@ public function viewer()
 
     public function index()
     {
-        return response()->json(LwtWhoops::all());
+        return response()->json(LwtWhoops::get());
```

Comment:
Unnecessary change, what style do we want to keep? Both statements are equal and thus both can be used.

Reply:
Doesn't really matter indeed, was messing around with some options and apparently ended up with this change. `all()` is just a nickname for `get()` though, which I think is kinda unnecessary in the first place. But I'm fine with either option